app/models/invoice.rb

```diff
@@ -1,4 +1,7 @@
 class Invoice < ApplicationRecord
   belongs_to :customer
   belongs_to :merchant
+  has_many :transactions
```

Comment:
I wonder if we should check with Josh about this. It makes sense with `has_many` if we're doing the black Thursday style, but i'm not sure yet if that's the paradigm we're working with for this.